apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -34,26 +34,42 @@ function wpcom_should_limit_global_styles( $blog_id = 0 ) {
 		}
 	}
 
-	// Do not limit Global Styles on sites created before we made it a paid feature (2022-12-15).
-	if ( $blog_id < 213403000 ) {
+	// Do not limit Global Styles on demo sites.
+	if ( wpcom_global_styles_has_blog_sticker( 'theme-demo-site', $blog_id ) ) {
 		return false;
 	}
 
-	if ( wpcom_site_has_feature( WPCOM_Features::GLOBAL_STYLES, $blog_id ) ) {
-		return false;
-	}
-
-	if ( function_exists( 'has_blog_sticker' ) && has_blog_sticker( 'theme-demo-site', $blog_id ) ) {
+	// Do not limit Global Styles on sites created before we made it a paid feature (2022-12-15),
+	// that had already used Global Styles.
+	if ( wpcom_premium_global_styles_is_site_exempt( $blog_id ) ) {
 		return false;
 	}
 
-	if ( function_exists( 'wpcomsh_is_site_sticker_active' ) && wpcomsh_is_site_sticker_active( 'theme-demo-site' ) ) {
+	// Do not limit Global Styles if the site paid for it.
+	if ( wpcom_site_has_feature( WPCOM_Features::GLOBAL_STYLES, $blog_id ) ) {
```

Comment:
Can we move this condition above `wpcom_premium_global_styles_is_site_exempt`? This way, the exempt logic is not executed on sites that have a valid subscription.